### DISCUSSION TITLE
Support Nested Chapters

### DISCUSSION THREAD

**Discussion by athendwyl** [PaulWoitaschek/Voice#1590] [Ideas]
Some audio formats, like matroska, support nested chapters. *The Way of Kings* by Brandon Sanderson, for example, broken up into parts, and interludes between those parts, and both the parts and interludes contain the *actual* chapters. There are also some chapters which are on the top level, like the Prologue, Pelude, and Epilogue.

Unfortunately, the way that the app works *currently* can only read the top level chapters, so when I open up the book, I can onle see "Prelude, Prologue, Part 1, Interludes, etc.", and none of the chapters within the parts or interludes. I would appreciate it if support for these nested chapters was added.

Note, I am aware of the problem with matroska described in #1018, but I have managed to get them working, which I've described in [this reply](https://github.com/PaulWoitaschek/Voice/issues/1018#issuecomment-1295766247) to that issue.

**Comment by PaulWoitaschek** [comment]
Could you post the output of ffprobe?

**Reply by athendwyl** [reply]
I can't figure out how to get the output of ffprobe to actually store in a file, and I can't figure out how to copy text from the terminal window, but I *can* tell you that *ffprobe* seems to only be able to detect top level chapters (the parts, not the chapters within them), so that's where the issue lies.

**Reply by PaulWoitaschek** [reply]
Then the only thing you can do for now is to open / find an issue in the ffmpeg repo. 
Btw you can pipe outputs. 

ffprobe file > my.logs
